e2e/playwright/support/server/default_config.ts

```diff
@@ -618,7 +622,7 @@ const defaultServerConfig: AdminConfig = {
                 Enable: true,
             },
             focalboard: {
-                Enable: true,
+                Enable: false,
             },
             playbooks: {
                 Enable: true,
```

Comment:
@saturninoabril 
just curious :
in screenshots updated below, playbook has disappeared and focal board is visible, but form flags it seems should be the other way around.

Reply:
Good question. Playbook has an issue with icon in the App bar which I'll address separately. Boards is using in-product and not plugin and that's the reason it's visible.

Reply:
This should be fixed now here - https://github.com/mattermost/mattermost-webapp/pull/12196